client/templates/admin/postEditor/postEditor.html

```diff
@@ -1,14 +1,12 @@
 <template name="postEditor">
-  <div class="panel panel-default">
+
     {{ #if isReady }}
 
       <div class="panel-heading">
         {{ #if isEditing }}
-          <h2 class="pull-left">Edit post</h2>
-          <button class="btn btn-primary pull-right savePost">Save post</button>
+          <button class="update-post-button savePost">Oppdater</button>
         {{ else }}
-          <h2 class="pull-left">New post</h2>
-          <button class="btn btn-primary pull-right savePost">Save new post</button>
+          <button class="add-btn savePost">Save new post</button>
```

Comment:
Cool, ping me when you review that and I merge, the rest is good to go. Good job @dagvonkr  :D